.github/workflows/test.yml

```
@@ -12,7 +12,7 @@ on:
   # request source branch is in the same repository.
   push:
     branches:
-      - "master"
+      - "next"
```

```suggestion
      - "master"
      - "next"
```
Run tests on both the master and next branches.